### ISSUE TITLE
Put json files into a folder?

### ISSUE THREAD

**Issue by jensneuse** [davegarvey/tyk-jenkins-cicd-sync#3]
Hey, really like what you're working on.
I wonder if putting the json files improves readability of the repo.
I usually like when you can see the repo README quite early without scrolling too much.

**Comment by davegarvey** [comment]
I agree, I've made a mess of the repo while I was figuring things out!

I'd like to fix the original Tyk-owned repo https://github.com/TykTechnologies/tyk-jenkins-cicd-example, then I can get rid of this one.
